README.md

```diff
@@ -55,3 +55,26 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 ## Setup a Trello Account and acquire a new 'Secret'
 This app uses Trello's API to fetch and save to-do tasks. In order to call their API, you need to first create an account - https://trello.com/signup
 Then generate an API key and token by following the instructions here - https://trello.com/app-key
+
+
+## Testing The app
```

Comment:
Minor, but we should also mention that this has a dependency on having Firefox installed (and Geckodriver if you do remove that from the checked in files)